src/main/java/org/spongepowered/api/world/ChunkTicketManager.java

```diff
@@ -188,6 +188,13 @@
          */
         int getMaxNumChunks();
 
+        /**
+         * Gets the world associated with this ticket during it's creation
+         *
+         * @return The world associated with this ticket
+         */
+        World getWorld();
```

Comment:
Recycled how? My issue is that the LoadingTicket is useless at the moment unless you arbitrarily apply a ticket to chunks when the callback is running since there's no NBT alternative for tickets yet and there's no way to check the world. I don't want to assign a ticket created with a World instance like the nether to a chunk that exists in the overworld.